### PR TITLE
Update Analytics pages in the docs

### PR DESCRIPTION
**Pull request by ericholscher**
This builds our properly feature pages for them,
and remove the old reference page that combined both.

Fixes https://github.com/readthedocs/readthedocs.org/issues/11587

---
:books: Documentation previews :books:

- User's documentation (`docs`): https://docs--11636.org.readthedocs.build/en/11636/

- Developer's documentation (`dev`): https://dev--11636.org.readthedocs.build/en/11636/

### CODE REVIEW

File: docs/user/traffic-analytics.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+   Traffic analytics demo
+
+Enabling Traffic analytics
+--------------------------
+
+#. Go to the new dashboard:
+
+    * `Community <https://app.readthedocs.org>`_
+    * `Business <https://app.readthedocs.com>`_
+
```
Comment (humitos) [inline docs/user/traffic-analytics.rst:30]:
```suggestion
#. Go to the new :term:`dashboard`:
```

I like more that pattern that uses the glossary and its definition to link to both of the dashboards. When you hover on the `dashboard` word, it shows you both of the links:

Peek 2024-10-02 12-51

I've been following it in other places and I'm happy with it.

Comment (ericholscher) [reply docs/user/traffic-analytics.rst:30]: Yea, that's a reasonable pattern until we build a better auto-redirect service 💯

File: docs/user/search-analytics.rst [hunk 2]
```diff
@@ -1,7 +1,18 @@
-How to use search analytics
-===========================
+Search Analytics
```
Comment (agjohnson) [inline docs/user/search-analytics.rst:1]:
```suggestion
Search analytics
```

File: docs/user/search-analytics.rst [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+================
 
-In this guide, you can learn to use Read the Docs' built-in search analytics for your documentation project.
+When someone visits your documentation and uses the built-in :doc:`server side search </server-side-search/index>` feature,
+Read the Docs will collect analytics on their search term.
```
Comment (agjohnson) [inline docs/user/search-analytics.rst:5]:
```suggestion
Read the Docs will collect analytics on their search queries.
```

File: docs/user/search-analytics.rst [hunk 4]
```diff
@@ -1,7 +1,18 @@
...
+
+Those are aggregated into a simple view of the
```
Comment (agjohnson) [inline docs/user/search-analytics.rst:7]:
```suggestion
These are aggregated into a simple view of the
```

File: docs/user/search-analytics.rst [hunk 5]
```diff
@@ -1,7 +1,18 @@
...
+"Top queries in the past 30 days".
+You can also download this data.
+
+This is helpful to optimize your documentation in alignment with your readers' interests.
+You can discover new trends and expand your documentation to new needs.
+
+Using Search Analytics
```
Comment (agjohnson) [inline docs/user/search-analytics.rst:14]:
```suggestion
Using search analytics
```

File: docs/user/search-analytics.rst [hunk 6]
```diff
@@ -39,3 +50,12 @@ The number of times the query has been used in a search is displayed as the **se
 In **Daily search totals**, you can see trends that might match special events in your project's publicity.
 If you wish to analyze these numbers in details, click :guilabel:`Download all data`
 to get a CSV formatted file with all available search analytics.
+
+Data storage
+------------
+
+The amount of analytics data stored for download depends which site you're using:
```
Comment (agjohnson) [inline docs/user/search-analytics.rst:57]:
```suggestion
The duration of analytics data stored for download depends on which site you're using:
```

File: docs/user/search-analytics.rst [hunk 7]
```diff
@@ -39,3 +50,12 @@ The number of times the query has been used in a search is displayed as the **se
...
+
+* On |org_brand|, the last 90 days are stored.
+* On |com_brand|, it goes from 30 to infinite storage
```
Comment (agjohnson) [inline docs/user/search-analytics.rst:60]:
```suggestion
* On |com_brand|, storage duration starts at 30 days and increases with plan level.
```

File: docs/user/search-analytics.rst [hunk 8]
```diff
@@ -39,3 +50,12 @@ The number of times the query has been used in a search is displayed as the **se
...
+   (check out `the pricing page <https://about.readthedocs.com/pricing/>`_).
```
Comment (agjohnson) [inline docs/user/search-analytics.rst:61]:
```suggestion
.. seealso::
   
    `Our plan pricing <https://about.readthedocs.com/pricing/>`_
        Compare our plan pricing and analytics storage duration.
```

File: docs/user/traffic-analytics.rst [hunk 9]
```diff
@@ -0,0 +1,43 @@
+Traffic Analytics
```
Comment (agjohnson) [inline docs/user/traffic-analytics.rst:1]:
```suggestion
Traffic analytics
```

File: docs/user/traffic-analytics.rst [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+=================
+
+Read the Docs aggregates statistics about visits to your documentation.
+This is mainly information about how often pages are viewed,
+and which return a `404 Not Found` error code.
+
+Traffic Analytics lets you see *which* documents your users are reading.
```
Comment (agjohnson) [inline docs/user/traffic-analytics.rst:8]:
```suggestion
Traffic analytics let you see *which* documents your users are reading.
```

File: docs/user/traffic-analytics.rst [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+This allows you to understand how your documentation is being used,
+so you can focus on expanding and updating parts people are reading most.
+
+To see a list of the top pages from the last month,
+go to the :guilabel:`Admin` tab of your project,
+and then click on :guilabel:`Traffic Analytics`.
```
Comment (agjohnson) [inline docs/user/traffic-analytics.rst:14]:
```suggestion
and then click on :guilabel:`Traffic analytics`.
```

File: docs/user/traffic-analytics.rst [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+
+.. figure:: /_static/images/addons-analytics.png
+   :width: 50%
+   :align: center
+   :alt: Traffic analytics demo
+
+   Traffic analytics demo
+
+Enabling Traffic analytics
```
Comment (agjohnson) [inline docs/user/traffic-analytics.rst:23]:
```suggestion
Enabling traffic analytics
```

File: docs/user/traffic-analytics.rst [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+
+#. Go to the new :term:`dashboard`:
+#. Click on a project name.
+#. Go to :guilabel:`Settings`, then in the left bar, go to :guilabel:`Addons`.
+#. Go to :guilabel:`Analytics` and check :guilabel:`Analytics enabled`.
+
+Data storage
+------------
+
+The amount of analytics data stored for download depends which site you're using:
```
Comment (agjohnson) [inline docs/user/traffic-analytics.rst:34]:
```suggestion
The duration of analytics data stored for download depends which site you're using:
```

File: docs/user/traffic-analytics.rst [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+
+* On |org_brand|, the last 90 days are stored.
+* On |com_brand|, it goes from 30 to infinite storage
```
Comment (agjohnson) [inline docs/user/traffic-analytics.rst:37]: Same edits as above here -- clarifying language here and using ``seealso`` block instead.

File: docs/user/traffic-analytics.rst [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+   (check out `the pricing page <https://about.readthedocs.com/pricing/>`_).
+
+
+.. seealso::
+
+   You can also access analytics data from :doc:`search results </search-analytics>`.
```
Comment (agjohnson) [inline docs/user/traffic-analytics.rst:43]: We've been using seealso with a definition list: https://dev.readthedocs.io/en/latest/style-guide.html#cross-references

This gives a way to link a page without working it into a sentence, which also gives more room to provide the user with _why the link is relevant_.

```suggestion
.. seealso::

    :doc:`Search analytics </search-analytics>`
        See what users are searching for in your documentation
```

**Comment by ericholscher** [comment]
I'm also going to merge this one, so it goes out in the stable release tomorrow 🎉

**Pull request merged**